app/assets/src/api/index.js

```diff
@@ -141,12 +141,9 @@ const uploadFileToUrlWithRetries = async (
     onUploadProgress
   };
 
-  // Set a 10s response timeout
-  const client = axios.create();
-  client.defaults.timeout = 10000;
```

Comment:
I experimented a little more and found that just leaving this timeout off was better. Otherwise I think it forced a timeout after 10 sec.